### PR TITLE
0.10.x into develop

### PR DESCRIPTION
**Pull request by benjaoming**
There were several conflicts, so I kindly request your help to solve them :)

Currently, there are .vue changes that are inconsistent with new changes in `develop` so I couldn't figure out how to solve the conflicts.

```
CONFLICT (content): Merge conflict in requirements/test.txt
CONFLICT (content): Merge conflict in kolibri/utils/env.py
CONFLICT (content): Merge conflict in kolibri/plugins/learn/assets/src/views/LearnExamReportViewer.vue
CONFLICT (content): Merge conflict in kolibri/plugins/coach/assets/src/views/exams/CreateExamPage/index.vue
CONFLICT (content): Merge conflict in kolibri/plugins/coach/assets/src/views/exams/CoachExamsPage/PreviewExamModal.vue
CONFLICT (content): Merge conflict in kolibri/plugins/coach/assets/src/views/exams/CoachExamReport/index.vue
CONFLICT (content): Merge conflict in kolibri/core/content/utils/import_export_content.py
CONFLICT (content): Merge conflict in kolibri/core/content/test/test_import_export.py
CONFLICT (content): Merge conflict in kolibri/core/content/management/commands/importcontent.py
CONFLICT (content): Merge conflict in kolibri/__init__.py
```

### CODE REVIEW

**Comment by benjaoming** [comment]
@jonboiser in order to solve these conflicts, please feel free to PR or push directly to my branch.

File: kolibri/core/content/test/test_import_export.py [hunk 1]
```diff
@@ -250,19 +274,19 @@ def test_local_import_oserror_dne(self, is_cancelled_mock, cancel_mock, path_moc
...
     @patch('kolibri.core.content.management.commands.importcontent.paths.get_content_storage_file_path')
     def test_local_import_oserror_permission_denied(self, path_mock, getsize_mock, logger_mock, annotation_mock):
-        dest_path = tempfile.mkstemp()[1]
-        path_mock.side_effect = [dest_path, '/test/dne']
-        getsize_mock.side_effect = ['1', OSError('Permission denied')]
-        with self.assertRaises(OSError):
-            call_command('importcontent', 'disk', self.the_channel_id, 'destination')
-            self.assertTrue('Permission denied' in logger_mock.call_args_list[0][0][0])
-            annotation_mock.assert_not_called()
+            dest_path = tempfile.mkstemp()[1]
```
Comment (lyw07) [inline kolibri/core/content/test/test_import_export.py]: Could you please unindent this part of test code by one? Thanks!

Comment (benjaoming) [reply kolibri/core/content/test/test_import_export.py]: Oh didn't see that, thanks!

**Comment by lyw07** [comment]
The python code looks good to me!

**Comment by benjaoming** [comment]
Thanks for checking, @lyw07 - Travis also says it's good :)

**Comment by benjaoming** [comment]
Great - more conflicts :)

**Comment by benjaoming** [comment]
@jonboiser @christianmemije @DXCanas  - just wanna flag that this needs someone with special insights to take lead on the merge conflicts, they are too complicated for others to understand :)

**Review by rtibbles — APPROVED** [review]

**Pull request merged**